release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1201,6 +1201,132 @@ For more information about the unsupported, community-maintained, version of the
 [id="ocp-release-note-cloud-compute-bug-fixes_{context}"]
 ==== Cloud Compute
 
+* When upgrading {gcp-short} clusters that use a boot disk that is not compatible with UEFI, you cannot enable shielded VM support. 
```

Comment:
```suggestion
* When upgrading {gcp-short} clusters that use a boot disk that is not compatible with UEFI, you cannot enable shielded-VM support. 
```
Does everyone know what a Unified Extensible Firmware Interface is?

Reply:
In this context, I think yeah